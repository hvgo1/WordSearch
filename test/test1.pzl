RHPIRNUMAE
HOALOIMSCT
YMTYHAHIRN
MOTRPUTMOO
EPETAQYISM
THRETNHLTA
IONOEIREII
CNKPMCTICD
SELBALLYSS
CSSTELPUOC

COUP LETS
DIA MONTE
META PHOR
RHY ME
SYLLA BLES