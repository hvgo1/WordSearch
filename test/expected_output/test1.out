COUPLETS (10, 10) (3, 10)
DIAMONTE (10, 8) (10, 1)
METAPHOR (5, 8) (5, 1)
RHYME (1, 1) (1, 5)
SYLLABLES (9, 9) (1, 9)
